src/streamlink/plugins/kanal7.py

```diff
@@ -0,0 +1,33 @@
+"""
+$description Turkish live TV channel.
```

Comment:
The description must include more information about the sites which that plugin supports, like who's the owner, etc. Take a look at other plugins.
https://en.wikipedia.org/wiki/Kanal_7